src/ui/component/commentCreate/view.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import { FormField, Form } from 'component/common/form';
+import Button from 'component/button';
+import ChannelSection from 'component/selectChannel';
+import { COMMENT_ACKNOWLEDGED, COMMENT_ACKNOWLEDGED_TRUE } from 'constants/settings';
+import { usePersistedState } from 'util/use-persisted-state';
+
+// props:
+type Props = {
+  uri: string,
+  claim: StreamClaim,
+  createComment: (string, string, string) => void,
+};
+
+export function CommentCreate(props: Props) {
+  const { createComment, claim } = props;
+  const { claim_id: claimId } = claim;
+  const [commentValue, setCommentValue] = usePersistedState(`comment-${claimId}`, '');
+  const [commentAck, setCommentAck] = usePersistedState(COMMENT_ACKNOWLEDGED, 'no');
+  const [channel, setChannel] = usePersistedState('COMMENT_CHANNEL', 'anonymous');
+
+  function handleCommentChange(event) {
+    setCommentValue(event.target.value);
+  }
+
+  function handleChannelChange(channel) {
+    setChannel(channel);
+  }
+
+  function handleCommentAck(event) {
+    setCommentAck(COMMENT_ACKNOWLEDGED_TRUE);
+  }
+  function handleSubmit() {
+    if (channel !== 'new' && commentValue.length) createComment(commentValue, claimId, channel);
+    setCommentValue('');
+  }
+
+  return (
+    <React.Fragment>
+      {commentAck !== COMMENT_ACKNOWLEDGED_TRUE && (
+        <section className="card card--section">
+          <div className="card__content">
+            <div className="media__title">About comments..</div>
+          </div>
+          <div className="card__content">
+            <div className="media__subtitle">Seriously, don&apos;t comment.</div>
```

Comment:
I'm not sure if we need a title along with this? I guess it depends on what that messaging ends up being.

Reply:
@kauffj was going to do some copy for this section.